docs/topics/api/reviewers.rst

```diff
@@ -84,6 +84,29 @@ add-on.
     :>json boolean needs_admin_content_review: Boolean indicating whether the add-on needs its content to be reviewed by an admin or not.
 
 
+-------------
+List Versions
+-------------
+
+This endpoint allows you to list versions that can be used either for :ref:`browsing <reviewers-versions-browse>` or diffing versions.
+
+    .. note::
+        Requires authentication and the current user to have ``ReviewerTools:View``
+        permission for listed add-ons as well as ``Addons:ReviewUnlisted`` for
+        unlisted add-ons. Additionally the current user can also be the owner
+        of the add-on.
+
+.. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/
+
+    :>json int id: The version id.
+    :>json string channel: The version channel, which determines its visibility on the site. Can be either ``unlisted`` or ``listed``.
+    :>json string version: The version number string for the version.
+    :>json string url: The (absolute) version detail URL.
```

Comment:
reminder that this will be dropped in #10698 so you should remove it from here too.

Reply:
Fixed in 3119c5f